gridstatus/tests/base_test_iso.py

```diff
@@ -78,9 +79,9 @@ def test_get_fuel_mix_historical_with_date_range(self):
         self._check_fuel_mix(df)
 
         # make sure right number of days are returned
-        assert df["Time"].dt.day.nunique() == num_days
+        assert df[time_column].dt.day.nunique() == num_days
 
-    def test_range_two_days_with_day_start_endpoint(self):
+    def test_get_fuel_mix_range_two_days_with_day_start_endpoint(self):
```

Comment:
I renamed this method and the one below to add `get_fuel_mix` to the name so it's easier to filter on these tests (e.g. running `pytest gridstatus/tests/test_ieso.py -k test_get_fuel_mix` to select all fuel mix related tests now works).